src/stores/useSocialMediaStore/useSocialMediaStore.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { act, renderHook } from '@testing-library/react';
+
+import * as useSocialMediaStoreHook from '~stores/useSocialMediaStore/useSocialMediaStore';
+
+vi.mock('~services/api/social-media/social-media', () => ({
+  SocialMediaService: {
+    fetch: vi.fn().mockResolvedValue([
+      { icon: 'discord-icon', id: 'DISCORD_EXAMPLE_ID', name: 'Discord' },
+      { icon: 'twitter-icon', id: 'TWITTER_EXAMPLE_ID', name: 'Twitter' },
+    ]),
+  },
+}));
+
+vi.mock('~services/api/accounts/accounts', () => ({
+  AccountsService: {
+    fetchAll: vi.fn().mockResolvedValue([
+      {
+        avatar: 'https://example.com/image1.jpg',
+        expiresAt: 'Fri May 17 2024 22:57:54 GMT-0300 (Brasilia Standard Time)',
+        generatedAt:
+          'Fri May 17 2024 22:57:54 GMT-0300 (Brasilia Standard Time)',
+        id: 1,
+        socialMediaId: 'DISCORD_EXAMPLE_ID',
+        token: 'DISCORD_EXAMPLE_TOKEN_1',
+        userName: 'Discord User 1',
+        valid: true,
+      },
+      {
+        avatar: 'https://example.com/image2.jpg',
+        expiresAt: 'Fri May 17 2024 22:57:54 GMT-0300 (Brasilia Standard Time)',
+        generatedAt:
+          'Fri May 17 2024 22:57:54 GMT-0300 (Brasilia Standard Time)',
+        id: 14,
+        socialMediaId: 'TWITTER_EXAMPLE_ID',
+        token: 'TWITTER_EXAMPLE_TOKEN_14',
+        userName: 'Twitter User 14',
+        valid: true,
+      },
+    ]),
+  },
+}));
+
+describe('useSocialMediaStore Hook', () => {
+  it('Checks if the fetchedSocialMediasMap is correctly set', async () => {
```

Comment:
acho que esse nome pode melhorar
"it('populates the `socialMedias` store from user accounts data', ...)"